src/bindings/python/requirements_test.txt

```diff
@@ -29,7 +29,7 @@ flake8_quotes
 mypy
 Pep8-naming
 pydocstyle
-pytest-forked
+pytest-forked; sys_platform == 'win32'
```

Comment:
is there any difference between `platform_system` and `sys_platform` ?

Reply:
Yes. `sys_platform` returns one of predetermined values:

- Linux - linux or linux2
- Windows - win32
- Windows/Cygwin - cygwin
- Windows/MSYS2 -  msys
- Mac OS X - darwin
- OS/2 - os2
- OS/2 EMX - os2emx
- RiscOS - riscos
- AtheOS - atheos
- FreeBSD 7 - freebsd7
- FreeBSD 8 - freebsd8
- FreeBSD N -  freebsdN
- OpenBSD 6 -  openbsd6
- AIX - aix

Whereas `platform_system` returns whatever [uname system call](https://pubs.opengroup.org/onlinepubs/9699919799/functions/uname.html) returns, which is a lot less predictable. I think we should stick to `sys_platform`.